nautobot_ssot/models.py

```diff
@@ -156,6 +168,12 @@ def get_source_url(self):
             kwargs={"class_path": self.job_result.job_model.class_path},
         )
 
+    def get_source_display(self):
+        """Display the name and link to the source worker associated with this instance."""
+        if self.get_source_url():
+            return format_html('<a href="{}">{}</a>', self.get_source_url(), self.source)
```

Comment:
Is this using 2 db calls since it's calling `get_source_url()` twice?
```suggestion
        source_url = self.get_source_url()
        if source_url:
            return format_html('<a href="{}">{}</a>', source_url, self.source)
```